runtimes/rust/lbr-prelude/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "lbr-prelude"
+version = "0.1.0"
+edition = "2021"
```

Comment:
2023 maybe? Just saying :)

Reply:
I just checked, the latest edition is 2021 (if this document is honest: https://doc.rust-lang.org/edition-guide/editions/index.html )

Reply:
omg I thought this is just for our package somehow, ignore me